<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCar</class>
 <widget class="QDialog" name="AddCar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>327</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>419</width>
    <height>327</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>419</width>
    <height>327</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Car Spreadsheet</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Images.qrc">
    <normaloff>:/icons/carIcon</normaloff>:/icons/carIcon</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #F4F1BB

</string>
  </property>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>290</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#F4F1BB;
	font: 8pt &quot;Yu Gothic UI Semilight&quot;;
}
QPushButton:pressed{
	background-color:rgb(255, 251, 195);
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okPushButton">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>290</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#F4F1BB;
	font: 8pt &quot;Yu Gothic UI Semilight&quot;;
}
QPushButton:pressed{
	background-color:rgb(255, 251, 195);
}</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>65</y>
     <width>105</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Yu Gothic UI Semilight&quot;;</string>
   </property>
   <property name="text">
    <string>Put a brand: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="brandLineEdit">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>70</y>
     <width>145</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #E6EBE0;
font: 12pt &quot;Yu Gothic UI Semilight&quot;;


</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="colorLineEdit">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>115</y>
     <width>145</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #E6EBE0;
font: 12pt &quot;Yu Gothic UI Semilight&quot;;


</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>110</y>
     <width>105</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Yu Gothic UI Semilight&quot;;</string>
   </property>
   <property name="text">
    <string>Put a color: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="priceLineEdit">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>165</y>
     <width>145</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #E6EBE0;
font: 12pt &quot;Yu Gothic UI Semilight&quot;;


</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>160</y>
     <width>105</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Yu Gothic UI Semilight&quot;;</string>
   </property>
   <property name="text">
    <string>Put a Price: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="powerLineEdit">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>215</y>
     <width>145</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #E6EBE0;
font: 12pt &quot;Yu Gothic UI Semilight&quot;;


</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>210</y>
     <width>105</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Yu Gothic UI Semilight&quot;;</string>
   </property>
   <property name="text">
    <string>Put a Power: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>20</y>
     <width>401</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Yu Gothic UI Semilight&quot;;</string>
   </property>
   <property name="text">
    <string>Enter all information about new car:</string>
   </property>
  </widget>
  <zorder>cancelPushButton</zorder>
  <zorder>okPushButton</zorder>
  <zorder>brandLineEdit</zorder>
  <zorder>label</zorder>
  <zorder>colorLineEdit</zorder>
  <zorder>label_2</zorder>
  <zorder>priceLineEdit</zorder>
  <zorder>label_3</zorder>
  <zorder>powerLineEdit</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
